src/js/controls.js

```diff
@@ -52,7 +56,13 @@
 				n.color = n._color
 		})
 		this.filter.undo().apply()
-		this._fire('reset')
+		this._fire('#reset')
```

Comment:
I think the `listing.js` file listens for the `reset` event so this probably breaks something